pallets/stableswap/src/weights.rs

```diff
@@ -181,6 +182,16 @@ impl WeightInfo for () {
 			.saturating_add(RocksDbWeight::get().reads(23_u64))
 			.saturating_add(RocksDbWeight::get().writes(7_u64))
 	}
+
+	fn remove_liquidity() -> Weight {
```

Comment:
maybe it is still WIP, but this weight function has not been added to runtime `pallet_swableswap.rs` weights, i guess the benchmark generation on the ref machine is still left?!

Reply:
will be rebenchamrked